client/extensions/woocommerce/state/sites/orders/selectors.js

```diff
@@ -93,3 +93,34 @@ export const getOrder = ( state, orderId, siteId = getSelectedSiteId( state ) )
 export const getTotalOrdersPages = ( state, siteId = getSelectedSiteId( state ) ) => {
 	return get( state, [ 'extensions', 'woocommerce', 'sites', siteId, 'orders', 'totalPages' ], 1 );
 };
+
+/**
+ * @param {Object} state Whole Redux state tree
+ * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
+ * @return {array} List of new orders.
+ */
+export const getNewOrders = ( state, siteId = getSelectedSiteId( state ) ) => {
+	// TODO: fetchOrders right now loads max number of orders, as pagination won't be supported until post v1
+	// We will possibly need to get this data another way once that behavior changes.
+	if ( ! areOrdersLoaded( state, 1, siteId ) ) {
```

Comment:
This is the only one I didn't address, since we are short circuiting the function here, rather then setting something (we only want to return `[]` if orders are not loaded)